ucla-prrla/src/app/institution/institution.component.html

```diff
@@ -25,9 +25,11 @@
                             <dt>Location: </dt>
                             <dd><strong>{{ memberInfo.prrla_member_location }}</strong></dd>
                             <dt>Phone: </dt>
-                            <dd><strong>{{ memberInfo.prrla_member_phone }}</strong></dd>
+                            <dd><strong><a href="tel:{{ memberInfo.prrla_member_phone }}">{{ memberInfo.prrla_member_phone }}</a></strong></dd>
                             <dt>Email: </dt>
                             <dd><strong><a href="mailto:{{ memberInfo.prrla_member_email }}">{{ memberInfo.prrla_member_email }}</a></strong></dd>
+                            <dt>Web contact: </dt>
+                            <dd><strong><a href="{{ memberInfo.prrla_member_web_contact }}">{{ memberInfo.prrla_member_web_contact }}</a></strong></dd>
```

Comment:
Maybe use a target value in the `href`, so the PRRLA page is still open/usable?

Reply:
Since it's such an easy changes: sure.

Reply:
Oh, nvm, I see the Website anchor tag has a blank target. 👍🏻

Reply:
✅